app/models/micropost.rb

```diff
@@ -4,7 +4,11 @@ class Micropost < ApplicationRecord
     length: {maximum: Settings.micropost.length.maximum}
   validates :user_id, presence: true
   validate  :picture_size
-  scope :load_by_user_id, ->user_id{where(user_id: user_id)}
+  scope :feed, ->user_id do
+    following_ids = Relationship.followed_ids_by_follower_id user_id
+    all_user_ids = following_ids + [user_id]
```

Comment:
`following_ids + [user_id]` lệnh này thì sao em

Reply:
thực ra em có thể truyền 2 tham số thay vì cộng Array..hihi
`scope :load_feed, ->(id, following_ids) do
    where "user_id IN (?) OR user_id = ?", following_ids, id
end`

Reply:
dạ